src/Form/CsvDataSourceType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of itk-dev/datatidy.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace App\Form;
+
+use App\Entity\CsvDataSource;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CsvDataSourceType extends AbstractDataSourceType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::getBaseFormBuilder($builder, $options);
```

Comment:
The name `getBaseFormBuilder` may be misleading, i.e. it's not obvious that calling `get…` actually has the side effect that fields are added to the form builder. Maybe we should let `AbstractDataSourceType` implement `buildForm`?